docs/standard/garbage-collection/implementing-disposeasync.md

```diff
@@ -124,13 +120,10 @@ In the preceding example, each asynchronous clean up operation is implicitly sco
 
 ### Unacceptable pattern
 
-The highlighted lines in the following code show what it means to have "stacked usings". If an exception is thrown from the `AnotherAsyncDisposable` constructor, neither object is properly disposed of. The variable `objTwo` is never assigned because the constructor did not complete successfully. As a result, the constructor for `AnotherAsyncDisposable` is responsible for disposing any resources allocated before it throws an exception.
+The highlighted lines in the following code show what it means to have "stacked usings". If an exception is thrown from the `AnotherAsyncDisposable` constructor, neither object is properly disposed of.
 
 :::code language="csharp" id="dontdothis" source="snippets/dispose-async/ExamplePatterns.cs" highlight="9-10":::
 
-> [!TIP]
-> Avoid this pattern as it could lead to unexpected behavior.
```

Comment:
Why is this being removed? This should stay.

Reply:
Please, this is eligible for removal since it gives **a tip** on phenomenon that is focused in whole chapter. I my casual day I would not appreciate **a tip** on anything being right 100 % obvious.

Reply:
➡️ Users frequently search for code, and then copy paste whichever result shows up without reading all the surrounding text. The callout here is intended to highlight (via color/page style) the fact that copying this result is problematic.

Reply:
Hmm… If I skip that right above is quite big heading **Unacceptable pattern** should not this be a note then?